### PR TITLE
add example on how to use `prisma init --preview-feature`

### PR DESCRIPTION
**Pull request by ruheni**
- Fixes a typo in the Preview features: `preview-features` -> `preview-feature`
- Adds example on how to use the `--preview-feature` flag
- Expand on description of the `--preview-feature` flag

### CODE REVIEW

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 1]
```diff
@@ -356,6 +356,20 @@ prisma init --datasource-provider sqlite
...
+prisma init --preview-feature multiSchema
+```
+
+The command sets `multiSchema` as a default Preview feature in your schema.
+
+```terminal
+prisma init --preview-feature multischema --preview-feature metrics
+```
+
+The command sets `multiSchema` and `metrics` as the default Preview features in your Prisma schema.
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
The command sets `multiSchema` and `metrics` as the Preview features in your Prisma schema.
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 2]
```diff
@@ -356,6 +356,20 @@ prisma init --datasource-provider sqlite
 
 The command output contains helpful information on how to use the generated files and begin using Prisma with your project.
 
+#### Run `prisma init --preview-feature`
+
+```terminal
+prisma init --preview-feature multiSchema
+```
+
+The command sets `multiSchema` as a default Preview feature in your schema.
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
The command sets `multiSchema` as a Preview feature in your schema.
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 3]
```diff
@@ -306,13 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                                                                              |                               |
+| `--generator-provider`  | No       | Define the default generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
+| `--preview-feature`     | No       | Define the default Preview features to use. [Learn more](/concepts/components/preview-features). To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--preview-feature`     | No       | Define the Preview features to use. [Learn more](/concepts/components/preview-features). To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
```
let's remove `default` everywhere, it's confusing

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 4]
```diff
@@ -306,13 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--datasource-provider` | No       | Specifies the default value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                                                       | `postgresql`                  |
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                                                                              |                               |
+| `--generator-provider`  | No       | Define the default generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 5]
```diff
@@ -306,13 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| Argument                | Required | Description                                                                                                                                                                                                                                                  | Default                       |
+| ----------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------- |
+| `--datasource-provider` | No       | Specifies the default value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                                                       | `postgresql`                  |
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--datasource-provider` | No       | Specifies the value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                                                       | `postgresql`                  |
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 6]
```diff
@@ -306,13 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--generator-provider`  | No       | Define the default generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
+| `--preview-feature`     | No       | Define the default Preview features to use. [Learn more](/concepts/components/preview-features). To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
+| `--output`              | No       | Specifies the default output location for the generated client. [Learn more](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path)                                                               | `node_modules/.prisma/client` |
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--output`              | No       | Specifies the output location for the generated client. [Learn more](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path)                                                               | `node_modules/.prisma/client` |
```

**Review by Jolg42 — APPROVED** [review]

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 7]
```diff
@@ -306,13 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
+| `--preview-feature`     | No       | Define the Preview features to use. [Learn more](/concepts/components/preview-features). To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
+| `--output`              | No       | Specifies the output location for the generated client. [Learn more](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path)                                                               | `node_modules/.prisma/client` |
```
Comment (ruheni) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| Argument                | Required | Description                                                                                                                                                                                                                                          | Default                       |
|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
| `--datasource-provider` | No       | Specifies the value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                                                       | `postgresql`                  |
| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                                                                      |                               |
| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
| `--preview-feature`     | No       | Define the Preview features to use. [Learn more](/concepts/components/preview-features). To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
| `--output`              | No       | Specifies the output location for the generated client. [Learn more](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path)                                                               | `node_modules/.prisma/client` |

```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 8]
```diff
@@ -356,6 +357,20 @@ prisma init --datasource-provider sqlite
...
+#### Run `prisma init --preview-feature`
+
+```terminal
+prisma init --preview-feature multiSchema
+```
+
+The command sets `multiSchema` as a Preview feature in your schema.
+
+```terminal
+prisma init --preview-feature multischema --preview-feature metrics
```
Comment (jharrell) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: The capitalization is different here compared to the above line. Is it `multischema` or `multiSchema` ?
```suggestion
prisma init --preview-feature multiSchema --preview-feature metrics
```

Comment (Jolg42) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: Better using `multiSchema` indeed

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 9]
```diff
@@ -306,13 +306,14 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                                                                      |                               |
+| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
+| `--preview-feature`     | No       | Define the Preview features to use. [Learn more](/concepts/components/preview-features). To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--preview-feature`     | No       | Define the [Preview features](/concepts/components/preview-features) to use. . To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 10]
```diff
@@ -306,13 +306,14 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--output`              | No       | Specifies the output location for the generated client. [Learn more](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path)                                                               | `node_modules/.prisma/client` |
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--output`              | No       | Specifies the [output location for the generated client](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path).                                                               | `node_modules/.prisma/client` |
```

**Review by janpio — APPROVED** [review]

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 11]
```diff
@@ -306,13 +306,14 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                                | `prisma-client-js`            |
+| `--preview-feature`     | No       | Define the [Preview features](/concepts/components/preview-features) to use. . To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
+| `--output`              | No       | Specifies the [output location for the generated client](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path).                                                               | `node_modules/.prisma/client` |
```
Comment (ruheni) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| Argument                | Required | Description                                                                                                                                                                                                                                | Default                       |
|-------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
| `--datasource-provider` | No       | Specifies the value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                                                                             | `postgresql`                  |
| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                                                            |                               |
| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                      | `prisma-client-js`            |
| `--preview-feature`     | No       | Define the [Preview features](/concepts/components/preview-features) to use. . To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
| `--output`              | No       | Specifies the [output location for the generated client](/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path).                                                                | `node_modules/.prisma/client` |
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 12]
```diff
@@ -306,13 +306,14 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                                                            |                               |
+| `--generator-provider`  | No       | Define the generator provider to use.                                                                                                                                                                                                      | `prisma-client-js`            |
+| `--preview-feature`     | No       | Define the [Preview features](/concepts/components/preview-features) to use. . To define multiple Preview features, you have to provide the flag multiple times for each Preview feature [See example](#run-prisma-init---preview-feature) |                               |
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--preview-feature`     | No       | Define the [Preview features](/concepts/components/preview-features) to use. To define multiple Preview features, you have to provide the flag multiple times for each Preview feature. [See example](#run-prisma-init---preview-feature) |                               |
```

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 13]
```diff
@@ -356,6 +357,20 @@ prisma init --datasource-provider sqlite
...
+prisma init --preview-feature multiSchema
+```
+
+The command sets `multiSchema` as a Preview feature in your schema.
+
+```terminal
+prisma init --preview-feature multiSchema --preview-feature metrics
+```
+
+The command sets `multiSchema` and `metrics` as the Preview features in your Prisma schema.
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: Should this maybe show the resulting `previewFeatures`?

Comment (ruheni) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: Done ✅

I've used the CodeWithResult component here.

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 14]
```diff
@@ -356,6 +356,58 @@ prisma init --datasource-provider sqlite
 
 The command output contains helpful information on how to use the generated files and begin using Prisma with your project.
 
+#### Run `prisma init --preview-feature`
+
+<CodeWithResult outputResultText="Prisma schema" expanded={true}>
+
+<cmd>```terminal prisma init --preview-feature multiSchema```</cmd>
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx:363]: This is broken and not displayed @ruheni 
image

Comment (ruheni) [reply content/400-reference/200-api-reference/200-command-reference.mdx:363]: Created a PR here: https://github.com/prisma/docs/pull/5410

**Pull request merged**